lib/cmd/fh3/admin/mbaas/update.js

```diff
@@ -1,15 +1,25 @@
 /* globals i18n */
 module.exports = {
-  'desc' : i18n._('Update an MBaaS.'),
-  'examples' : [{ cmd : 'fhc admin mbaas update --id=<MBaaS id> --url=<MBaaS URL> --servicekey=<MBaaS Service Key> --username=<MBaaS User Name> --password=<MBaaS Password> --decoupled=<bool> --size=<MBaaS Size> --label=<MBaaS label> --editable=<bool>', desc : 'Updates an mbaas with id <mbaasId>'}],
+  'desc' : i18n._('Update an MBaaS Target.'),
+  'examples' :
+    [{
+      cmd : 'fhc admin mbaas update --id=<MBaaS id> --url=<MBaaS URL> --servicekey=<MBaaS Service Key> --username=<MBaaS User Name> --password=<MBaaS Password> --decoupled=<bool> --size=<MBaaS Size> --label=<MBaaS label> --editable=<bool>',
+      desc : 'Updates FeedHenry MBaaS Target with id <mbaasId>'
+    },
+    {
+      cmd : 'fhc admin mbaas update --id=<MBaaS id> --url=<OpenShift Master URL> --routerDNSUrl=<OpenShift Router DNS> --servicekey=<MBaaS Service Key> --fhMbaasHost=<MBaaS URL> --label=<MBaaS label> --editable=<bool>',
+      desc : 'Updates OpenShift 3 MBaaS Target with id <mbaasId>'
+    }],
```

Comment:
I think maybe if we are not documenting it we can just remove it from the example, what do you think?